Stratis.Bitcoin/Features/BlockStore/BlockStoreLoopStep.cs

```diff
@@ -35,21 +35,16 @@ internal void SetNextStep(BlockStoreLoopStep step)
         /// <param name="disposeMode">This will <c>true</c> if Flush() was called on the BlockStore</param>
         /// <param name="cancellationToken">Cancellation token to check</param>
         /// <returns>BlockStoreLoopStepResult</returns>
-        internal async Task<BlockStoreLoopStepResult> Execute(ChainedBlock nextChainedBlock, bool disposeMode, CancellationToken cancellationToken)
+        internal async Task<StepResult> Execute(ChainedBlock nextChainedBlock, bool disposeMode, CancellationToken cancellationToken)
         {
-            var result = BlockStoreLoopStepResult.Next();
-
             foreach (var step in this.steps)
             {
                 var stepResult = await step.ExecuteAsync(nextChainedBlock, cancellationToken, disposeMode);
-                if (stepResult.ShouldBreak || stepResult.ShouldContinue)
-                {
-                    result = stepResult;
-                    break;
-                }
+                if (stepResult == StepResult.Continue || stepResult == StepResult.Stop)
```

Comment:
just a note fore the future - we intend to make a new code style policy that will include parentheses to be used as:

```
if ((stepResult == StepResult.Continue) || (stepResult == StepResult.Stop))
```

not a problem now as the new rules are not yet formed, but might be easier to comply already (see https://github.com/stratisproject/StratisBitcoinFullNode/issues/258) for more